lib/hammer_cli/output/output.rb

```diff
@@ -9,11 +9,11 @@ def initialize(context={}, options={})
     attr_accessor :default_adapter
 
     def print_message(msg, msg_params={})
-      adapter.print_message(msg.to_s, msg_params)
+      adapter.print_message(msg.to_s, msg_params) if @context[:verbosity].to_i > 0
```

Comment:
I think that we should enable specifying the verbosity per message (i.e. pass the verbosity as an argument instead of hardcoding 0).